### PR TITLE
Removes unnecessary lines of description and makes things more like they are in Civ5.

### PR DESCRIPTION
**Pull request by DW8030**
(You might want to revert the change to the Polish ability)

### CODE REVIEW

File: jsons/Buildings.json [hunk 1]
```diff
@@ -78,8 +80,9 @@
         "gold": 2,
         "specialistSlots": {"Merchant": 1},
         "hurryCostModifier": 15,
-        "uniques": ["[+5]% [Production] [in this city]",
-            "[+25]% [Gold] [in this city]","Requires a [Market] [in this city]"],
+	  "percentStatBonus": {"gold": 25},
+	  "uniques": ["[+5]% [Production] [in this city]"],
+	  "requiredBuilding": "Market",
```
Comment (ravignir) [inline jsons/Buildings.json:85]: why this change tho? requiredBuilding and percentStatBonus are an old way of doing it.

File: jsons/Units.json [hunk 2]
```diff
@@ -111,8 +111,8 @@
 		"requiredTech": "Metallurgy",
 		"obsoleteTech": "Combined Arms",
 		"upgradesTo": "Anti-Tank Gun",
-		"uniques": ["Can move after attacking","No defensive terrain bonus",
-			"[-33]% Strength <vs cities>","Consumes [1] [Horses]"],
+		"requiredResource": "Horses",
```
Comment (ravignir) [inline jsons/Units.json:114]: requiredResource is an old way of doing it.

Comment (ravignir) [reply jsons/Units.json:114]: it is old, but still working.

File: jsons/Buildings.json (hunk 1 above)
Comment (DW8030) [reply jsons/Buildings.json:85]: See here my proof: https://github.com/yairm210/Unciv/blob/master/android/assets/jsons/Civ%20V%20-%20Gods%20%26%20Kings/Buildings.json (last accessed 23/03/2022 16:14).

File: jsons/Units.json (hunk 2 above)
Comment (DW8030) [reply jsons/Units.json:114]: Can you show me something that says the new way is to add it as unique?

Comment (ravignir) [reply jsons/Units.json:114]: https://github.com/yairm210/Unciv/pull/5920

Comment (ravignir) [reply jsons/Units.json:114]: Uniques are also used - and the only error i was having had to do with Venice ability.

requiredResource etc. will be obsolete sooner or later.

Comment (DW8030) [reply jsons/Units.json:114]: You can use uniques, but requiredResource etc. are not (yet) outdated, with (as far I've seen) no set date when they will become outdated (if at all), and until that point it's (in my opinion) plain ugly to read "Consumes 1 Horses" in the description of a Cavalry-replacement that self-explanatory consumes horses (if it doesn't, it is shown as "Horses not required"). That description is entirely redundant. Do you find it to be visually appealing? (if that's true, then you of course must not merge those changes)
